fissa/core.py

```diff
@@ -199,6 +200,12 @@ def __init__(self, images, rois, folder=None, nRegions=4,
             :class:`~extraction.DataHandlerTifffile`.
             Note: if `datahandler` is set, the `lowmemory_mode` parameter is
             ignored.
+        verbosity : int, optional
+            Indicates the level of verbosity. The levels are:
+            0 - No outputs
+            1 - Progress bars + high level summary
+            2 - Print intermediate progress steps
+            3 - Print per-cell progress
```

Comment:
This doesn't render correctly when the docs are generated. When you're doing more complicated things like this, I find it is a good idea to locally generate and view the documentation (`make -C docs html; sensible-browser docs/_build/html/index.html`) to check how it is rendered.

Also it needs to say when the new argument was added.

You might want to use a list:
```suggestion
            Indicates the level of verbosity. The levels are:

            - ``0``: No outputs
            - ``1``: Progress bars and high level summary
            - ``2``: Print intermediate progress steps
            - ``3``: Print per-cell progress

            .. versionadded:: 1.0.0

```
or a definition list:
```suggestion
            Indicates the level of verbosity. The levels are:

            0
                No outputs
            1
                Progress bars and high level summary
            2
                Print intermediate progress steps
            3
                Print per-cell progress

            .. versionadded:: 1.0.0

```